### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by ginabeki**
- Deleted all the boilerplate from CRA (text, images, styles).
- Created a directory called components and added a new Calculator.js file.
- In Calculator.js, I created a React component that matches the required design

### Screenshot
calc

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      <div className="item item-1">0</div>
+      <div className="item item-2">AC</div>
+      <div className="item item-3">+/-</div>
+      <div className="item item-4">%</div>
+      <div className="item item-5 bg">÷</div>
+      <div className="item item-6">7</div>
+      <div className="item item-7">8</div>
+      <div className="item item-8">9</div>
+      <div className="item item-9 bg">x</div>
+      <div className="item item-10">4</div>
```
Comment (emmiiorji) [inline src/components/Calculator.js]: As mentioned in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/e404f92ae7b446ece2b5081c85779f0d9ab59731/math-magicians/project_components.md?plain=1#L40), kindly ensure that all components are class-based.

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,10 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
```
Comment (emmiiorji) [inline src/App.js]: Just as pointed out above and as specified in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/e404f92ae7b446ece2b5081c85779f0d9ab59731/math-magicians/project_components.md?plain=1#L40), kindly make all components to be class-based.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @ginabeki, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

A little bit more

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Adhered to Gitflow


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**